Extension/Configuration/Models/SorterConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using JetBrains.Annotations;
+
+namespace YAPO.Configuration.Models
+{
+    [UsedImplicitly]
+    public class SorterConfiguration
+    {
+        public SortDirection SortDirection;
+
+        public bool UpgradableOnTop;
+
+        public bool SortOrderOpposite;
+
+        public SortMode CurrentSortByMode;
+
+        public SortMode CurrentThenByMode;
```

Comment:
Could get rid of the new lines in between to use the same style as other data classes in the code base